examples/tree/panel.js

```diff
@@ -61,7 +61,8 @@ Ext.application({
         });
 
         treeStore = Ext.create('GeoExt.data.TreeStore', {
-            layerGroup: olMap.getLayerGroup()
+            layerGroup: olMap.getLayerGroup(),
+            showLayerGroupNode: true
```

Comment:
What do you think of names like `inclusive`, `includeRoot`, or even `rootVisible`, which we have at the treepanel? Just thinking out loud here. Could we possibly even inherit this from a panel we live in?